include/variant_detection/variant_output.hpp

```diff
@@ -4,26 +4,34 @@
 
 #include <seqan3/std/filesystem>
 
-#include "structures/cluster.hpp"     // for class Cluster
+#include "iGenVar.hpp"              // for cmd_arguments
+#include "structures/cluster.hpp"   // for class Cluster
 
 
 /*! \brief Detects genomic variants from junction clusters and prints them to output stream in VCF format.
  *
  * \param[in]       clusters    - input junction clusters
+ * \param[in]       args        - command line arguments:\n
+ *                                **args.min_var_length** - minimum length of variants to detect - *default: 30 bp*
  * \param[in, out]  out_stream  - output stream
  *
  * \details Extracts genomic variants from given junction clusters.
  */
-void find_and_output_variants(std::vector<Cluster> const & clusters, std::ostream & out_stream);
+void find_and_output_variants(std::vector<Cluster> const & clusters,
+                              cmd_arguments const & args,
+                              std::ostream & out_stream);
```

Comment:
Yeah, I thought about that for a while, too, and ended up passing the args because this method handles the actual variant calling and output. In the future, we might add a lot of arguments that need to be passed to the method, e.g.
- maximum variant length
- maximum tolerated inserted sequence size for deletions (currently hard-coded as 5)
- maximum tolerated distance between breakends for insertions (current hard-coded as 1)
- variant classes to output
- output of symbolic or sequences alleles
- ...

Reply:
okay, thats a good argument. Thanks for adding some of these variables directly :)